test_graphyte.py

```diff
@@ -141,6 +142,38 @@ def test_timestamp_generated(self):
         timestamp = int(match.group(1))
         self.assertTrue(send_time - 2 <= timestamp <= send_time + 2)
 
+    def test_send_socket_do_raise_error(self):
+
+        class SenderWithError(graphyte.Sender):
+            def __init__(self, *args, **kwargs):
+                graphyte.Sender.__init__(self, 'dummy_host', *args, **kwargs)
```

Comment:
I don't think we need to override `__init__` here, right? Just pass in `dummy_host` as you're doing anyway. Same below.